core/templates/dev/head/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -43,6 +43,8 @@ oppia.factory('CollectionLinearizerService', [
       });
     };
 
+    // Given a non linear collection input, the function will linearize it by
+    // picking the first node it encounters on the branch and ignore the others.
     var _getCollectionNodesInPlayableOrder = function(collection) {
```

Comment:
NB: This one is kept because it would otherwise conflict with ``getCollectionNodesInPlayableOrder()``